### ISSUE TITLE
Not working in v7.8.4 (64 bit)

### ISSUE THREAD

**Issue by toddhd** [emmetio/npp#28]
The directions stated that I could simply search the Plugins Admin for the Emmet plugin, however it was not there. So I took the second step(s) and tried to install it manually. The files are there but NotePad++ doesn't see it. There is no Emmet option under plugins, and when I search for Emmet in the Plugins Admin, it doesn't see it. Not sure what to do?

**Comment by dpbarrett** [comment]
@toddhd, I moved the EmmetNPP.dll into the EmmetNPP folder as per instructions I found @ [https://npp-user-manual.org/docs/plugins/#install-plugin-manually](https://npp-user-manual.org/docs/plugins/#install-plugin-manually) and it worked for me.

GL

**Comment by YakovL** [comment]
yeah, would be nice to be able to install the plugin via the plugin manager (I'm on 7.8.4, too)

**Comment by clfaulkner** [comment]
NPP gives a 32/64 bit error. The extension is evidently 32bit and it won't install on a 64bit NPP installation.

![NPP Err Msg 2020-04-08](https://user-images.githubusercontent.com/36777713/78775815-7a0f5a00-795c-11ea-86d8-43fed14398de.png)

**Comment by chengeast** [comment]
I get same err too! What can I do?

**Comment by CennoxX** [comment]
Duplicate of #19
